### PR TITLE
feat: wishlist page and responsive navbar

### PR DESCRIPTION
**Pull request by ParthParolekar**
Added wishlist page and made the navbar responsive

### CODE REVIEW

File: Components/navbar/navbar.css [hunk 1]
```diff
@@ -23,17 +24,56 @@
...
   .big-logo {
     display: block;
     height: 5rem;
     margin-left: 2rem;
   }
 
+  #hamburger {
+    display: none;
+  }
+
```
Comment (partha8) [inline Components/navbar/navbar.css:68]: avoid using id selectors. It will cause problems in the future.

**Review by partha8 — APPROVED** [review]
Everything looks good!

File: Components/navbar/navbar.css (hunk 1 above)
Comment (ParthParolekar) [reply Components/navbar/navbar.css:68]: Done

File: Components/navbar/navbar.css [hunk 2]
```diff
@@ -23,17 +24,56 @@
...
+  z-index: 1;
+  transition: 0.3s all;
+}
+
+.nav-links a {
+  margin: 0 1rem;
+}
+
+.nav-links-not-visible {
+  top: -180px;
```
Comment (venkateshwarans) [inline Components/navbar/navbar.css]: I think translate could be a better alternative here. 
read more about it - https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/

File: index.html [hunk 3]
```diff
@@ -230,5 +231,7 @@ <h3>Popular</h3>
         </div>
       </section>
     </main>
+
+    <script src="./Components/navbar/navbar.js"></script>
```
Comment (venkateshwarans) [inline index.html:235]: avoid capitalizing folder names.

**Review by venkateshwarans — APPROVED** [review]
Looks great overall. Evolved so much  :)

File: index.html (hunk 3 above)
Comment (ParthParolekar) [reply index.html:235]: I will take care of it from now

File: Components/navbar/navbar.css (hunk 2 above)
Comment (ParthParolekar) [reply Components/navbar/navbar.css]: Made the required changes.

**Comment by ParthParolekar** [comment]
Thank you @partha8 and @venkateshwarans for the PR reviews. Merging now

**Pull request merged**